plutus-core/plutus-core/src/PlutusCore/TypeCheck/Internal.hs

```diff
@@ -139,8 +131,10 @@ lookupBuiltinM
     :: (AsTypeError err term uni fun ann, HasTypeCheckConfig cfg uni fun, Ix fun)
     => ann -> fun -> TypeCheckM uni fun cfg err (Normalized (Type TyName uni ()))
 lookupBuiltinM ann fun = do
-    BuiltinTypes mayArr <- view $ tceTypeCheckConfig . tccBuiltinTypes
-    case mayArr >>= preview (ix fun) of
+    BuiltinTypes arr <- view $ tceTypeCheckConfig . tccBuiltinTypes
+    -- Believe it or not, but 'Data.Array' doesn't seem to expose any way of indexing into an array
```

Comment:
I didn't believe it, but you seem to be right.  Maybe that's because `Data.Array` seems to have been there forever and they don't want to change it?

Reply:
Dunno. Probably just nobody cares.